tests/test_transformer/test_object_transformer.py

```diff
@@ -45,19 +46,40 @@ def setUp(self) -> None:
         tr = ObjectTransformer()
         tr.source_schemaview = SchemaView(str(PERSONINFO_SRC_SCHEMA))
         tr.target_schemaview = SchemaView(str(PERSONINFO_TGT_SCHEMA))
-        tr.load_transformer_specification(PERSONINFO_TR)
-        self.tr = tr
+        tr.specification = yaml_loader.load(
+            str(PERSONINFO_TR), target_class=TransformationSpecification
+        )
+        self.spec = yaml.safe_load(open(str(PERSONINFO_TR)))
+        self.target_data = yaml.safe_load(open(str(PERSONINFO_TGT_DATA)))
+        self.target_object = yaml_loader.load(str(PERSONINFO_TGT_DATA), target_class=tgt_dm.Agent)
```

Comment:
load up the expected data structure to make it easier to comprehensively check the results of the transformations